File: C++/day2.cpp
#include <cmath>
#include <cstdio>
#include <iostream>

using namespace std;

int main() {
    double M, T, X;
    cin >> M >> T >> X;
    int price = (int) round(M * (1 + (T + X) / 100));
    printf("The final price of the meal is $%d.", price);
    return 0;
}

File: Java/day2.java
public class Arithmetic {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double M = sc.nextDouble(); // original meal price
        int T = sc.nextInt(); // tip percentage
        int X = sc.nextInt(); // tax percentage

        double gross = M * (1 + T / 100.0 + X / 100.0);
        int total = (int) Math.round(gross);

        System.out.printf("The final price of the meal is $%d.", total);
    }
    
}
